demo/Main.py

```diff
@@ -2697,6 +2697,7 @@ def main():
     except:
         pass
     app = MyApp(False)
+    locale = wx.Locale(wx.LANGUAGE_DEFAULT)
```

Comment:
What problem did you have that necessitated this change?

Reply:
I can't run demo.py in Idle under Windows/Python2.7 unless the locale is set.
Otherwise it throws the exception and the demo never runs. Full text:
```
wx._core.wxAssertionError: C++ assertion "strcmp(setlocale(LC_ALL, NULL), "C") == 0" failed at ..\..\src\common\intl.cpp(1579) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.
Things are going to break, please only change locale by creating wxLocale objects to avoid this!
```